scripts/test/Muon/load_file_widget/loadfile_model_test.py

```diff
@@ -6,14 +6,15 @@
 # SPDX - License - Identifier: GPL - 3.0 +
 import os
 import unittest
+from mantidqt.utils.qt.testing import GuiTest
 
 import six
 
 from Muon.GUI.Common.load_file_widget.model import BrowseFileWidgetModel
 from Muon.GUI.Common.test_helpers.context_setup import setup_context_for_tests
 
 
-class LoadFileWidgetModelTest(unittest.TestCase):
+class LoadFileWidgetModelTest(GuiTest):
```

Comment:
why do we need to use GUITest here?

Reply:
The notify calls emitted when the model is changed go via an invoke method to ensure that any resulting updates to the GUI happen on the GUI thread. This invoke method requires the QApplication to exist.